<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-image: url(:/backgroundImg/img/def-background.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTaskTab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string> 主界面</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="TaskGroup" name="MainSubject" native="true">
          <layout class="QGridLayout" name="gridLayout_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_importFile"/>
    <addaction name="action_importMdFile"/>
   </widget>
   <widget class="QMenu" name="menu_addTaskInCurGroup">
    <property name="title">
     <string>添加任务</string>
    </property>
    <addaction name="action_addTaskInCurrentGroup"/>
   </widget>
   <widget class="QMenu" name="menu_addGroup">
    <property name="title">
     <string>添加任务组</string>
    </property>
    <addaction name="action_addNewGroup"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>更改...</string>
    </property>
    <addaction name="action_changeSubjectName"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>移除</string>
    </property>
    <addaction name="action_removeCurSubjectTargetTask"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置任务字体</string>
    </property>
    <addaction name="action_setFontColor"/>
    <addaction name="action_setFontSize"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
   <addaction name="menu_addTaskInCurGroup"/>
   <addaction name="menu_addGroup"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_addTaskInCurrentGroup">
   <property name="text">
    <string>添加当前主题下的任务</string>
   </property>
  </action>
  <action name="action_addNewGroup">
   <property name="text">
    <string>添加主题</string>
   </property>
  </action>
  <action name="action_changeSubjectName">
   <property name="text">
    <string>更换当前主题名称</string>
   </property>
  </action>
  <action name="action_removeCurSubjectTargetTask">
   <property name="text">
    <string>移除任务</string>
   </property>
  </action>
  <action name="action_setFontColor">
   <property name="text">
    <string>设置字体颜色</string>
   </property>
  </action>
  <action name="action_setFontSize">
   <property name="text">
    <string>设置字体大小</string>
   </property>
  </action>
  <action name="action_importFile">
   <property name="text">
    <string>导入文件</string>
   </property>
  </action>
  <action name="action_importMdFile">
   <property name="text">
    <string>导入MD文件</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaskGroup</class>
   <extends>QWidget</extends>
   <header location="global">taskgroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
